fili-core/src/test/groovy/com/yahoo/bard/webservice/web/filters/HealthCheckFilterSpec.groovy

```diff
@@ -166,7 +166,7 @@ class HealthCheckFilterSpec extends Specification {
     def "Healthy /data check passes"() {
         setup:
         HealthCheckFilter filter = Spy(HealthCheckFilter)
-        filter.isHealthy() >> { true }
+        filter.getFirstUnhealthy() >> { true }
```

Comment:
Interestingly enough, it doesn't. Once the tests are made to work (silly logic bugs...), there's some cast exceptions that show up. Fix coming shortly.